src/main/java/com/minecolonies/coremod/entity/ai/basic/AbstractEntityAIBasic.java

```diff
@@ -627,7 +637,7 @@ private AIState waitForPickaxe()
         return IDLE;
     }
 
-    /**
+   /**
```

Comment:
This format is wrong, the space is intended there ;) Have you imported the intellij formatter?